ddh-com/copilot/ddh-com/manifest.yml

```diff
@@ -0,0 +1,69 @@
+# The manifest for the "ddh-com" service.
+# Read the full specification for the "Load Balanced Web Service" type at:
+#  https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/
+
+# Your service name will be used in naming your resources like log groups, ECS services, etc.
+name: ddh-com
+type: Load Balanced Web Service
+
+# Distribute traffic to your service.
+http:
+  # Requests to this path will be forwarded to your service.
+  # To match all requests you can use the "/" path.
+  path: '/'
+  # You can specify a custom health check path. The default is "/".
+  # healthcheck: '/'
+  healthcheck:
+    path: '/'
+    success_codes: '200'
+    healthy_threshold: 3
+    unhealthy_threshold: 10
+    interval: 160s
+    timeout: 120s
+
+# Configuration for your containers and service.
+image:
+  # Docker build arguments. For additional overrides: https://aws.github.io/copilot-cli/docs/manifest/lb-web-service/#image-build
+  build: ../Dockerfile
+  # Port exposed through your container to route traffic to it.
+  port: 3838
+
+cpu: 1024       # Number of CPU units for the task.
+memory: 8192   # Amount of memory in MiB used by the task.
+count: 1       # Number of tasks that should be running in your service.
+exec: true     # Enable running commands in your container.
+
+# Optional fields for more advanced use-cases.
+#
+#variables:                    # Pass environment variables as key value pairs.
+#  LOG_LEVEL: info
+variables:
+   AWS_DEFAULT_REGION: us-east-1
+   AWS_SQS_SERVICE_URL: https://sqs.us-east-1.amazonaws.com/344055253315/emailer
+   AWS_SQS_BUCKET_ID: ddh-sqs-logs
+   AWS_REPORT_BUCKET_ID: ddh-reports
+   AWS_DATA_BUCKET_ID: ddh-data
+   AWS_DATA_PRIVATE_BUCKET_ID: ddh-data-private
```

Comment:
we no longer have ddh-data-private

<img width="1218" alt="image" src="https://github.com/heurekalabsorg/ddh_app/assets/36932093/25d03e1f-6fbf-4140-aca2-8376e2fad084">